markets/dagstore/miner_api.go

```diff
@@ -201,7 +201,7 @@ func (m *minerAPI) GetUnpaddedCARSize(ctx context.Context, pieceCid cid.Cid) (ui
 		return 0, xerrors.Errorf("no storage deals found for piece %s", pieceCid)
 	}
 
-	len := pieceInfo.Deals[0].Length
+	l := pieceInfo.Deals[0].Length
 
-	return uint64(len), nil
+	return uint64(l), nil
```

Comment:
Maybe remove the intermediate `l` and just `return uint64(pieceInfo.Deals[0].Length)`